### PR TITLE
HW 2 Exercise 5 | Form (WIP)

### PR DESCRIPTION
**Pull request by hickamt**
# Form | Console Out
I have completed a working version of the form input which will console.table() the from field input.

# Feedback
I had difficulty with the form resetting on 'submit'. To resolve this I invoke the 'reset' button in the JS Function.
If you can think of another reason that is causing this I would like to handle this using the form + submit button only.
Also, any other comments or helpful hints are appreciated.

### CODE REVIEW

File: hw2/05-form.html [hunk 1]
```diff
@@ -10,10 +10,113 @@
...
+            for="email"
+            aria-label="enter your email"
+            >Email:</label
+          >
+          <input
+            class="form-control"
+            type="email"
+            id="email"
+            name="email"
+            min="10"
```
Comment (linh224) [inline hw2/05-form.html]: Hi Todd,
I think this is incorrect with **min="10"** because this is the context of an email input field.

**Review by linh224 — COMMENTED** [review]
Hi Todd,
When I performed Wave testing, it reported 1 error "1 X Missing form label" and 1 arlet "Orphaned form label." It's fantastic that you've finally fixed it. Aside from that, I believe that exercise 5 works as requirements. Your coding also teaches me a lot. Thank you very much!

File: hw2/05-form.html (hunk 1 above)
Comment (hickamt) [reply hw2/05-form.html]: I have removed the function call that clears the user input.

File: hw2/05-form.js
Comment (JoshuaWachana) [inline hw2/05-form.js:1]: This is fine, but next time possibly consider using form.elements as shown in [Chapter 18 of the JS book](https://eloquentjavascript.net/18_http.html) to query the form data. I personally find it easier to use.

File: hw2/05-form.js [hunk 2]
```diff
@@ -1 +1,62 @@
...
+  } catch (error) {
+    console.log(error);
+  }
+});
+
+const highlightWords = function highlightWords() {
+  const words = document.querySelectorAll("p");
+  for (let word of words) {
+    if (word.textContent.includes("JavaScript")) {
+      word.style.color = "red";
```
Comment (JoshuaWachana) [inline hw2/05-form.js:59]: Since bootstrap is being used another option would be to use the "bg-danger" class.

**Pull request merged**